app/[id]+api.ts

```diff
@@ -1,79 +1,86 @@
-import { deleteGuest, getGuest, Guest } from '../database/guests';
+import { deleteGuestInsecure, getGuestInsecure } from '../database/guests';
+import { ExpoApiResponse } from '../ExpoApiResponse';
+import { Guest, guestsSchema } from '../migrations/00000-createTableGuests';
+
+type ExpoResponseBodyGet =
```

Comment:
naming here (check one of the Next.js Route Handlers) - we name it after the entity, not including "Expo"

Reply:
Changed it to `GuestReponseBody..`

[cb39815](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/4/commits/cb39815b331bbc8df2ef7e4ff95b26775aebfcdc)